pilot/util/checksum.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
Ok to add this module, but please see the checksum functions in pilot/util/filehandling - calculate_checksum(), calculate_adler32_checksum(), calculate_md5_checksum(), get_checksum_value(), get_checksum_type().We should move all of these functions into checksum.py and update the usages.